web/src/features/Dashboard/api/useNextAppVersion.ts

```diff
@@ -0,0 +1,66 @@
+import { useRef, useState, useEffect } from "react";
```

Comment:
This file probably looks familiar since I just copy pasta'd it. We can refine this later.